### DISCUSSION TITLE
Support Virtual threads in jdk sender

### DISCUSSION THREAD

**Discussion by zettelmj** [open-telemetry/opentelemetry-java#6480] [Q&A]
As far as I can see it is currently not possible to use virtual threads in the jdk sender exporter.
I don't see any easy way to enable a switch in the current code base.
So the question(s) I have are:
Is using virtual threads for the exporter actually desirable (I think it is)
What would be the best/acceptable way. 

Jens

**Comment by jkwatson** [comment]
I agree. This would be a great option. Implementation will require some thought, as the BatchSpanProcessor, and our exporters would need some tweaking to make this possible.

**Comment by zettelmj** [comment]
I will give it a try. Once I've figured out how to get gradle to cooperate on compiling with language level 21...
